cms/views.py

```diff
@@ -44,7 +45,12 @@ class PostViewSet(viewsets.ModelViewSet):
     queryset = Post.objects.all()
     serializer_class = PostSerializer
     permission_classes = (PostPermission,)
-    filterset_fields = ['sites', ]
+    filter_backends = [DjangoFilterBackend,
+                       filters.SearchFilter, filters.OrderingFilter]
+    filterset_fields = ['sites']
```

Comment:
skusal som vcera search, filter a ordering na cms/post - vyzeralo to v pohode, akurat `sites` filter mi siel iba pre strom. FE WIP tu: https://github.com/ZdruzenieSTROM/webstrom-frontend/pull/510

skusim tento tyzden este tie competitions modely, ci je vsetko ok. do pagination mozno este zabiehat nebudem, uvidim.

Reply:
No neviem mne toto normalne funguje: `http://localhost:8000/cms/post/?sites=2` Nemas nejake divne dáta?

Reply:
vies co, rebasoval som to, lebo boli nejake posledne zmeny ohladom sites... tak moze byt. skusim neskor.
nebude to tymi zmenami ovplyvnene ale? mozno by si mohol rebasnut tiiez

Reply:
my bad, mal som tam este nejaky zbytok client-side filtru, ktory mi to pokazil. okej, minimalne na Postoch teda search, filter, aj ordering funguje 👌